plugin/pkg/shoot/validator/admission.go

```diff
@@ -1124,7 +1138,13 @@ func getDefaultMachineImage(machineImages []core.MachineImage, imageName string,
 		}
 	}
 
-	latestMachineImageVersion, err := helper.DetermineLatestMachineImageVersion(validVersions, true)
+	// TODO(acumino): remove this once supported version of arm machine image are available
```

Comment:
Why we need this? Because you internally add new arm images as `preview`? You can promote your images after time to `supported` and then the defaulting will work. We don't want to add such hacks in the Gardener codebase.